### DISCUSSION TITLE
Rename api folder

### DISCUSSION THREAD

**Discussion by elHornair** [nuxt/nuxt#18068] [Questions] [answered]
According to the [docs](https://v3.nuxtjs.org/docs/directory-structure/server), everything in the `/server/api` folder is automatically read for generating API endpoints. Is it possible do rename / reconfigure this, so everything in `/server/foobar` is treated as API endpoint?

**Comment by wobsoriano** [comment, marked as answer]
I think there's currently no option for this. What you can do for now is to add it in `serverMiddleware` in `nuxt.config.ts`.

```ts
export default defineNuxtConfig({
  serverMiddleware: [
    { path: "/foobar/users", handler: "~/server/foobar/users.ts" },
    { path: "/foobar/products", handler: "~/server/foobar/products.ts" },
  ],
})
```

**Reply by elHornair** [reply]
Perfect, thank you! 🙌

**Reply by mihail727** [reply]
But is it possible to somehow change `nuxt.config` so that in the address bar instead of `localhost:3000/api/` it can be renamed to `localhost:3000/server/`?
